src/unstract/sdk/x2txt.py

```diff
@@ -59,8 +68,16 @@ def process(
         output_file_path: Optional[str] = None,
         **kwargs: dict[Any, Any],
     ) -> str:
-        return self.x2text_instance.process(input_file_path, output_file_path, **kwargs)
+        return self._x2text_instance.process(
+            input_file_path, output_file_path, **kwargs
+        )
 
-    @deprecated("Use the class instance")
+    @deprecated(
+        "Deprecated method. Use the latest X2Text to "
+        "instantiate and then invoke process()"
```

Comment:
@gaya3-zipstack I believe `deprecated` library will throw warning that it is deprecated, we do not need to mention again.

In that case let's change the message to `Use process() method of X2Text instance.`.

Reply:
Done